static/app/views/replays/detail/network/networkTableCell.tsx

```diff
@@ -156,14 +173,19 @@ const Cell = styled('div')<{
   hasOccurred: boolean | undefined;
   hasOccurredAsc: boolean | undefined;
   isCurrent: boolean;
+  isDetailsOpen: boolean;
   isHovered: boolean;
   isStatusError: boolean;
   numeric?: boolean;
+  onClick?: any;
 }>`
   display: flex;
   align-items: center;
   padding: ${space(0.75)} ${space(1.5)};
   font-size: ${p => p.theme.fontSizeSmall};
+  cursor: ${p => (p.onClick ? 'pointer' : 'inherit')};
+
+  font-weight: ${p => (p.isDetailsOpen ? 'bold' : 'inherit')};
```

Comment:
This is mostly a demonstration of how we can style things. It's functional, but we might want to do something else to show that the row is open.